engine/api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/ProcessState.scala

```diff
@@ -94,6 +94,8 @@ final case class AllowDeployStateStatus(name: String) extends StateStatus
 
 final case class NotEstablishedStateStatus(name: String) extends StateStatus
 
+final case class UndeterminedStateStatus(name: String) extends StateStatus
+
```

Comment:
`override def isUndetermined: Boolean = true` and we can change `ps.status.isInstanceOf[NotEstablishedStateStatus])` for `ps.isUndetermined`?

Reply:
but `NotEstablishedStatus` means is not undetermined. It would make no sense... We can tell it is not established, cause we know exactly what status it is..? Or do I understand it wrong?

Reply:
You're right.. :)